### PR TITLE
Create getCITask.py

### PR DESCRIPTION
**Pull request by istairbn**
No synthetic changes, just the underlying task. 

Answers this issue #44 

Gets the underlying XML or json. We use this a lot!

### CODE REVIEW

File: src/main/resources/xlr_xldeploy/getCITask.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+from xlr_xldeploy.XLDeployClientUtil import XLDeployClientUtil
+
+xld_client = XLDeployClientUtil.create_xldeploy_client(xldeployServer, username, password)
+
+test = xld_client.check_ci_exist(ciID)
```
Comment (jdewinne) [inline src/main/resources/xlr_xldeploy/getCITask.py]: Can you put throw_on_fail as an argument?

File: src/main/resources/xlr_xldeploy/getCITask.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+if throwOnFail and not test:
+    raise Exception(ciID + " does not exist")
```
Comment (jdewinne) [inline src/main/resources/xlr_xldeploy/getCITask.py]: When using it as an argument, you don't need to throw your own exception anymore.

**Review by jdewinne — COMMENTED** [review]
Can you also add the synthetic changes?

File: src/main/resources/xlr_xldeploy/getCITask.py (hunk 1 above)
Comment (istairbn) [reply src/main/resources/xlr_xldeploy/getCITask.py]: Done

File: src/main/resources/xlr_xldeploy/getCITask.py (hunk 2 above)
Comment (istairbn) [reply src/main/resources/xlr_xldeploy/getCITask.py]: Indeed - thrown it out!

**Pull request merged**